src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxResolvingClassLoader.java

```diff
@@ -39,9 +37,9 @@
             return this.getClass().getClassLoader().loadClass(name);
         } else {
             ClassLoader parentLoader = getParent();
-            Class<?> c = load(parentClassCache, name, parentLoader, () -> {
+            Class<?> c = load(parentClassCache, name, parentLoader, item -> {
                 try {
-                    return Optional.of(parentLoader.loadClass(name));
+                    return Optional.of(parentLoader.loadClass(item));
```

Comment:
Not clear from reading sources without an IDE handy what `item` is going to be here. Is there a reason this cannot stay as `name`?

Reply:
Name collision with local variable `name`.  And if we're going to pass in a string value we should use it, right?

Reply:
Uh, sure, might just be nice to document that this is an alias for `name.

Reply:
@jglick Okay, cleaned up again.